tests/config/test_training_model.py

```diff
@@ -11,3 +11,11 @@ def test_training_wrong_values_by_assignments(minimum_training: dict):
     training.num_epochs = 2
     with pytest.raises(ValueError):
         training.num_epochs = -1
+
+    training.max_steps = 2
+    with pytest.raises(ValueError):
+        training.max_steps = 0
```

Comment:
This definitely will not pass. That might be the test that is currently failing in the PR!